docs/vlab/overview.md

```diff
@@ -71,6 +74,23 @@ INFO: /dev/kvm exists
 KVM acceleration can be used
 ```
 
+### ORAS
+For convenience HedgeHog provides a script to install oras:
+```bash
+curl -fsSL https://i.hhdev.io/oras | bash
+```
```

Comment:
````suggestion
### ORAS

For convenience Hedgehog provides a script to install oras:

```bash
curl -fsSL https://i.hhdev.io/oras | bash
```
````

I don't think we spell `HedgeHog`.

Also regarding the case - Is it `ORAS` (heading) or `oras` (paragraph)?

Reply:
so, I think when talking about the project we spell it as ORAS which matches how the project itself spells it. When using it as a title, Oras, and when referring to the cli command, `oras` or however it would be typed into a shell to execute.